libs/durability/disk_failure_recovery_libs.py

```diff
@@ -272,12 +272,14 @@ def get_user_data_space_in_bytes(self, master_obj: LogicalNode, memory_percent:
             return True, 0
 
     @staticmethod
-    def perform_near_full_sys_writes(s3userinfo, user_data_writes, bucket_prefix: str) -> tuple:
+    def perform_near_full_sys_writes(s3userinfo, user_data_writes, bucket_prefix: str,
```

Comment:
Can you move this function out to another fresh python module?  I/O to near full system is generic and is not linked to just recovery cases.

Reply:
Yes, we are planning to work on Lib refactoring, will take it as an improvement for next PR.